decimal64.go

```diff
@@ -37,12 +37,12 @@ type Decimal64 struct {
 
 // decParts stores the constituting decParts of a decimal64.
 type decParts struct {
```

Comment:
Is there a section that would explain to me what each one does by any chance? e.g. a source of truth